pylint_odoo/test_repo/broken_module/models/broken_model.py

```diff
@@ -30,8 +30,11 @@ class TestModel(models.Model):
     )
 
     def my_method1(self, variable1):
-        #  Shouldn't show error of field-argument-translate
+        "Shouldn't show error of field-argument-translate"
         self.my_method2(_('hello world'))
 
     def my_method2(self, variable2):
+        '''
```

Comment:
But in this case you are checking the beginning of the docstring with an empty line, not the correct quote. Expected errors should be then one more?